app/client/styles/components/_button.scss

```diff
@@ -18,14 +18,18 @@ input[type="button"] {
 	}
 
 	&:hover,
-	&:focus {
+	&.focused {
```

Comment:
@SeanMcGrath I removed the focus rules because I don't think they're useful right now and they mostly result in a button already looking selected when we switch between rooms.